projects/fsastorefrontstyles/scss/components/account/_premium-calendar.scss

```diff
@@ -65,6 +65,24 @@
         }
       }
     }
+    .icon-triangle-bottom {
+      position: absolute;
+      right: 5%;
+      top: 5%;
+      @include fs-transition;
+      @include media-breakpoint-down(md) {
+        color: var(--cx-color-6);
+        @include vertical-center;
```

Comment:
Shouldn't this mixin be included in any breakpoint instead only for mobile? Is there a real need for `top: 5%;`? `vertical-center` should handle that?

Reply:
Text in the table is not vertically centered, because of that there is difference in top attribute on the larger and mobile screens. In the case I use vertical-center for larger screens arrow won't be aligned with the text.

Reply:
Yes, but arrow should be centered in relation to accordion, not the text. Even if you try to align it to text, that would be a very hard thing to achieve for every case, since some text may break into several lines, and arrow would be fixed to 5%. And then it would stiill look off...